docs/TEAM.md

```diff
@@ -61,6 +61,8 @@
 | | [Melissa Huerta](https://github.com/piratelicorne) | Hi! I'm Systems Engineer from Perú, 3+ yrs of experience in IT Infrastructure, I have knowledge in C and Javascript. I recently started in Development as Flutter Developer, currently working on side projects 👩🏽‍💻 and blogging for #WomenWhoCodeSV                                                                         |
 | | [Abdou Masoudi](https://github.com/uplancer)       | a selft-taught Web Developer.|
 | | [Viraj](https://github.com/viraj-patil)            | I'm a full stack developer from Pune, India. Total 9+ years of experience in software industry as a developer. |
+| <img src="https://avatars2.githubusercontent.com/u/56103269?s=60&v=4"> | [Saurabh Srivastava](https://github.com/vasudeveloper001) | From India, 18 months Software Development Experience mainly in backend PHP development. Know about JavaScript, HTML, CSS, Nodejs, Python basics as well. Learning JAVA currently and starting a YouTube Channel on coding.
+=======
```

Comment:
@vasudeveloper001  Verify your changes in the preview mode.

![image](https://user-images.githubusercontent.com/72256599/97099724-41de2780-16b2-11eb-92b4-ee2cd7c3279f.png)

Reply:
What should I do here?

Reply:
Add "**|**" at the last and then preview the changes in the VSCode and github. It will fix formatting issue.

![image](https://user-images.githubusercontent.com/72256599/97100874-2bd66400-16be-11eb-920f-6009b4766edc.png)